packages/client/src/components/MemoryGameCard/MemoryGameCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { theme as t, CardProps, Card, Icon } from '@codement/ui';
+
+export interface MemoryGameCardProps extends CardProps {
+    flipped?: boolean;
+    state?: 'correct' | 'incorrect';
+    content?: string;
+}
+
+const StyledCard = styled(Card)<MemoryGameCardProps>`
+  position: relative;
+  width: 160px;
+  height: 160px;
+  left: 0%;
+  top: 0%;
+
+  background: ${t.color('white')};
+  box-shadow: 0px 2px 10px ${t.color('grey.200')};
+  border-radius: ${t.borderRadius.large};
+
+  align-items: center;
+
+  & > span {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
For this, we can use the `centerAbsolute` found [here](https://github.com/code-mentoring/learn/blob/f1bdcab02b0b5308a786914abdd64abc72b76eda/packages/ui/css/common.ts#L3)

Reply:
cool! changed